src/components/forms/cbr/CbrZoneForm.js

```diff
@@ -0,0 +1,208 @@
+            className="textInputWide"
+            name="description"
+            value={this.state.description}
+            labelText={"Description"}
+            onChange={this.handleInputChange}
+            invalid={
+              this.state.description.length < 0 ||
+              this.state.description.length > 300
+            }
+            invalidText={
+              "Invalid description, must be between 0 and 300 characters."
+            }
+            enableCounter={true}
+          />
+        </IcseFormGroup>
+        {/* show subforms if not modal*/}
+        {this.props.isModal !== true && (
+          <>
+            {/* Addresses */}
+            <IcseFormTemplate
+              name="Addresses"
+              subHeading
+              addText="Create an Address"
+              arrayData={this.props.data.addresses}
+              innerForm={CbrExclusionAddressForm}
+              disableSave={this.props.addressProps.disableSave}
+              onDelete={this.props.addressProps.onDelete}
+              onSave={this.props.addressProps.onSave}
+              onSubmit={this.props.addressProps.onSubmit}
+              propsMatchState={this.props.propsMatchState}
+              innerFormProps={{ ...addressInnerFormProps }}
+              hideAbout
+              toggleFormProps={{
+                hideName: true,
+                submissionFieldName: "cbr_zones",
+                disableSave: this.props.addressProps.disableSave,
+                type: "formInSubForm",
+              }}
+            />
+            {/* exclusions */}
+            <IcseFormTemplate
+              name="Exclusions"
+              subHeading
+              addText="Create an Exclusion"
+              arrayData={this.props.data.exclusions}
+              innerForm={CbrExclusionAddressForm}
+              disableSave={this.props.exclusionProps.disableSave}
+              onDelete={this.props.exclusionProps.onDelete}
+              onSave={this.props.exclusionProps.onSave}
+              onSubmit={this.props.exclusionProps.onSubmit}
+              propsMatchState={this.props.propsMatchState}
+              innerFormProps={{ ...exclusionInnerFormProps }}
+              hideAbout
+              toggleFormProps={{
+                hideName: true,
+                submissionFieldName: "cbr_zones",
+                disableSave: this.props.exclusionProps.disableSave,
+                type: "formInSubForm",
+              }}
+            />
+          </>
+        )}
+      </div>
+    );
+  }
+}
+
+CbrZoneForm.defaultProps = {
+  data: {
+    name: "",
+    description: "",
+    account_id: "",
+    addresses: [],
+    exclusions: [],
+  },
+  isModal: false,
+};
+
+CbrZoneForm.propTypes = {
+  addressProps: PropTypes.shape({
+    disableSave: PropTypes.func,
+    onDelete: PropTypes.func,
+    onSave: PropTypes.func,
+    onSubmit: PropTypes.func,
+  }),
+  data: PropTypes.shape({
+    name: PropTypes.string.isRequired,
+    account_id: PropTypes.string,
```

Comment:
is this account_id not required?

Reply:
nope - only required within addresses or exclusions https://registry.terraform.io/providers/IBM-Cloud/ibm/latest/docs/resources/cbr_zone